en-us/configuration.md

```diff
@@ -0,0 +1,204 @@
+      load_balance_algorithm: RandomWeight
+      # array of data sources
+      data_sources:
+        - name: employees-master
+          # The weight of the data source, r means read, w means write, and the following numbers represent the weight
+          weight: r0w10
+        - name: employees-slave
+          weight: r10w0
+  - name: redirect3
+    # The shd mode stands for sharding, that is, sharding database and sharding table, which can solve the problems of distributed transactions, read-write splitting, sharding database and sharding table
+    mode: shd
+    config:
+      # db group array, a db group represents a set of primary and secondary databases of a sharded database.
+      db_groups:
+        # The drug database is split into two databases, drug_0 and drug_1.
+        - name: drug_0
+          load_balance_algorithm: RandomWeight
+          data_sources:
+            - name: drug_0
+              weight: r10w10
+        - name: drug_1
+          load_balance_algorithm: RandomWeight
+          data_sources:
+            - name: drug_1
+              weight: r10w10
+      # logic_tables describes the logic of database sharding        
+      logic_tables:
+        # drug represents the name of the database before the sharding. After the sharding, it will be distinguished by an underscore and a number after the noun, such as drug_0, drug_1
+        - db_name: drug
+          # drug_resource is the name of the logical table, which is used in the user business to query, and it will be rewritten to the actual table name during actual execution
+          table_name: drug_resource
+          # Whether to allow full table scans, that is, whether to allow requests to be executed on all table shards
+          allow_full_scan: true
+          # sharding rule
+          sharding_rule:
+            # column represents the sharding key
+            column: id
+            # Sharding algorithm, currently only the sharding algorithm based on digital modulo is implemented, namely NumberMod
+            sharding_algorithm: NumberMod
+          # topology is used to describe the topology of the logical table
+          topology:
+            "0": 0-4
+            "1": 5-9          
+```
+
+The above code shows three executor configurations of DBPack, which are `sdb`, `rws`, `shd` three modes. In `shd` mode, there are two sharded databases `drug_0` and `drug_1` indicating that the `drug` database is split into 2 databases. The `topology` in the above code indicates that the `drug_resource` table is split into 10 sharded tables, which belong to two databases, namely:
+
+```
+drug_0: drug_resource_0、drug_resource_1、drug_resource_2、drug_resource_3、drug_resource_4
+drug_1: drug_resource_5、drug_resource_6、drug_resource_7、drug_resource_8、drug_resource_9
+```
+
+After calculating which shard table needs to execute the sql request based on the shard key, it can be determined which physical DB should be executed according to the topology.
```

Comment:
```suggestion
After calculating which shard table needs to execute the sql request based on the shard key, it can determine which physical DB should be executed according to the topology.
```